gateway-discovery-cm/src/main/java/org/apache/knox/gateway/topology/discovery/cm/ClouderaManagerServiceDiscoveryMessages.java

```diff
@@ -142,6 +142,12 @@ void failedToInstantiateJAASConfigurationFileImplementation(String implementatio
            text = "Terminating monitoring of {1} @ {0} for configuration changes because there are no referencing descriptors.")
   void stoppingConfigMonitoring(String discoverySource, String clusterName);
 
+  @Message(level = MessageLevel.WARN, text = "Missing property in previously saved service discovery configuration {0}")
+  void missingServiceDiscoveryConfigProperty(String propertyName);
+
+  @Message(level = MessageLevel.DEBUG, text = "There is no cluster configuration to check yet.")
```

Comment:
There aren't any format string parameters in the message, but the method has 2 arguments.

Reply:
Nice catch; fixed!